generate/cmd/libs/go2idl/client-gen/BUILD

```diff
@@ -0,0 +1,49 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I've read that bazelfiles in vendor tend not to play nice with bazel.  Have you confirmed this works?